.mergify.yml

```diff
@@ -2,6 +2,7 @@
 pull_request_rules:
   - name: Merge PRs that are ready
     conditions:
+      - status-success=continuous-integration/travis-ci/pr
```

Comment:
I copied it from TemplateControl without realising there was no check of Travis status. 

This is not needed for TemplateControl because there is not tests in there. :-(